metals/src/main/scala/scala/meta/internal/builds/ScalaCliBuildTool.scala

```diff
@@ -27,7 +32,10 @@ class ScalaCliBuildTool(
     createBspFileArgs(workspace).map(systemProcess).getOrElse {
       // fallback to creating `.bsp/scala-cli.json` that starts JVM launcher
       val bspConfig = workspace.resolve(".bsp").resolve("scala-cli.json")
-      bspConfig.writeText(ScalaCli.scalaCliBspJsonContent())
+      languageClient
+        .showMessage(new MessageParams(MessageType.Info, "scala-cli bspConfig"))
```

Comment:
Could we just log it or show in status bar? I don't think it's really useful to show this and make the user need to close it. Especially since it can obscure some other window part.

Reply:
We can show it in the status bar. Normally when a config is generated using `BspConfigGenerator.runUnconditionally` there is a `slowTask` and info in the status bar. I felt like we should show something here for consistency.

Reply:
Sure, but since we don't show anything for quick actions I think it's fine to only show it in the status bar